webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -6390,7 +6390,7 @@ var userOverride = {
                     success: function (data) {
                         var isMessages = true;
                         //Display/hide lead(email) collection template
-                        CURRENT_GROUP_DATA.createdAt = data.groupFeeds[0].createdAtTime;
+                        CURRENT_GROUP_DATA.createdAt = data && data.groupFeeds[0].createdAtTime;
```

Comment:
data &&
                            data.groupFeeds[0] &&

same as the code below it at line no. 6395